File: Java/src/jakopec/model/Entitet.java
package jakopec.model;

public abstract class Entitet {

    private int id;

    public Entitet(int id) {
        this.id = id;
    }

    public Entitet() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

}

File: PHP/src/Jakopec/Entitet.php
<?php


namespace Jakopec;


abstract class Entitet
{

    private $id;

    /**
     * Entitet constructor.
     * @param int $id
     */
    public function __construct(int $id = 0)
    {
        $this->id = $id;
    }


    /**
     * @return int
     */
    public function getId(): int
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId(int $id)
    {
        $this->id = $id;
    }


}